src/Features/OnboardingAutomateTaxes.php

```diff
@@ -55,9 +55,10 @@ public static function on_onboarding_profile_completed() {
 			$wcs_tos_accepted = \WC_Connect_Options::get_option( 'tos_accepted' );
 		}
 
-		if ( $jetpack_connected && $wcs_version && $wcs_tos_accepted ) {
+		if ( $jetpack_connected && $wcs_version && $wcs_tos_accepted && self::automated_tax_is_supported() ) {
```

Comment:
Not directly related to the issue in this PR, but I imagine not many sites will be Jetpack connected prior to setup so this will rarely get used right?  Is this mostly for Atomic sites?

Reply:
@joshuatf 

> Not directly related to the issue in this PR, but I imagine not many sites will be Jetpack connected prior to setup so this will rarely get used right? Is this mostly for Atomic sites?

I don't know if it was designed for Atomic sites but your intuition sounds right imo that this will rarely get used in any other scenario.